docs/source/blog/movement-v0_0_21.md

```diff
@@ -0,0 +1,82 @@
+---
+blogpost: true
+date: Dec 5, 2024
+author: Niko Sirmpilatze
+location: London, England
+category: release
+language: English
+---
+
+# Release v0.0.21 and next steps
+
+_This is our inaugaural blogpost, containing a summary of the `v0.0.21` release and a preview of what's coming next in 2025._
+
+## What's new in movement v0.0.21?
+
+:::{tip}
+See our [installation guide](target-installation) for instructions on how to
+install the latest version or upgrade from an existing installation.
+:::
+
+__Input/Output__
+
+- We have added the {func}`movement.io.load_poses.from_multiview_files` function to support loading pose tracking data from multiple camera views.
+- We have made several small improvements to reading bounding boxes tracks. See our new {ref}`example <sphx_glr_examples_load_and_upsample_bboxes.py>` to learn more about working with bounding boxes.
+- We have added a new {ref}`example <sphx_glr_examples_convert_file_formats.py>` on using `movement` to convert pose tracking data between different file formats.
+
+__Kinematics__
+
+The {mod}`kinematics <movement.kinematics>` module has been moved from `movement.analysis.kinematics` to `movement.kinematics` and packs a number of new functions:
+- {func}`compute_forward_vector <movement.kinematics.compute_forward_vector>`
+- {func}`compute_head_direction_vector <movement.kinematics.compute_head_direction_vector>`
+- {func}`compute_pairwise_distances <movement.kinematics.compute_pairwise_distances>`
+- {func}`compute_speed <movement.kinematics.compute_speed>`
+- {func}`compute_path_length <movement.kinematics.compute_path_length>`
+
+__Breaking changes__
+
+- We have dropped support for using filtering and
+kinematic functions via the `move` accessor syntax,
+because we've found the concept hard to convey to new users. All functions are henceforth solely accessible by importing them from the relevant modules. Having one way of doing things should reduce the maintenance burden on our end. See an example below:
```

Comment:
```suggestion
because we've found the concept hard to convey to new users. All functions are henceforth solely accessible by importing them from the relevant modules. Having one way of doing things simplifies the mental model for users and reduces the maintenance effort on our side. See an example below:
```
Just adding the benefit for users too.